src/renderer.cpp

```diff
@@ -705,8 +705,8 @@ void Renderer::bindTextures()
                             if (internalState.bDoMirrorTextureWrap) flags |= BGFX_SAMPLER_U_MIRROR | BGFX_SAMPLER_V_MIRROR | BGFX_SAMPLER_W_CLAMP;
 
                             if (internalState.bIsMovie) flags |= BGFX_SAMPLER_U_CLAMP | BGFX_SAMPLER_V_CLAMP | BGFX_SAMPLER_W_CLAMP;
-
-                            if (!internalState.bDoTextureFiltering || !internalState.bIsExternalTexture) flags |= BGFX_SAMPLER_MIN_POINT | BGFX_SAMPLER_MAG_POINT | BGFX_SAMPLER_MIP_POINT;
+                            
+                          if (!internalState.bDoTextureFiltering) flags |= BGFX_SAMPLER_MIN_POINT | BGFX_SAMPLER_MAG_POINT | BGFX_SAMPLER_MIP_POINT;

```

Comment:
Please revert this line to what it was before, this PR shouldn't touch the renderer at all. It took us quite an effort to get it working for both FF7 and FF8. Thanks

Reply:
But the renderer itself is what's hard blocking filtering of internal textures.

The pull request depends on me stopping this hard override.  If i don't do this, nothing else in mu pull request has any effect at all.

There's is literally no way to override this check. It was the entire reason turning bilinear filtering on did nothing before unless you were modding.